Java-Data-Types.md

```diff
@@ -0,0 +1,148 @@
+# Java data-Types
+
+Java is a strongly ­typed language. This means that, in Java, each data type has its own strict definition. There are no implicit data type conversions when any conflicts occur between the data types. Any change in data types should be explicitly declared by the programmer.
+ 
+Java defines 8 simple data types. They are divided into the following categories:
```

Comment:
When you are saying 8 data types, using a cardinal number; better to just list down the 8 types, instead of creating sub-classes.